src/parser/jobs/drg/modules/Positionals.js

```diff
@@ -177,7 +177,7 @@ export default class Positionals extends Module {
 			return <Table.Row key={combo.time}>
 				<Table.Cell>{this.createTimelineButton(combo.time)}</Table.Cell>
 				<Table.Cell><ActionLink {...action} /></Table.Cell>
-				<Table.Cell textAlign="center">{this._checkIcon(combo.trueNorthCharges > 0, '')} (<Plural id="drg.positionals.tn-charges" value={combo.trueNorthCharges} one="# charge" other="# charges" />)</Table.Cell>
+				<Table.Cell>{this._checkIcon(combo.trueNorthCharges > 0, '')} (<Plural id="drg.positionals.tn-charges" value={combo.trueNorthCharges} one="# charge" other="# charges" />)</Table.Cell>
```

Comment:
oh no my center text alignment D:
Unless there's a design reason to change this, I'd suggest leaving it centered.

Reply:
It looks really awkward as it is right now, so I did that to fix the alignment!